src/Modal.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Modal, { setAppElement } from './Modal';
+
+beforeAll(() => {
+    document.documentElement.innerHTML = '<div id="app" data-testid="app" />';
+    window.scroll = jest.fn();
+
+    setAppElement('#app');
+});
+
+it('should render its content when open and not render after closed', () => {
+    const { queryByText, rerender } = render(<Modal isOpen>Hello World</Modal>);
+
+    expect(queryByText('Hello World')).toBeTruthy();
```

Comment:
I usually assert like this: `expect(queryByText('Hello World')).toBeInTheDocument();`

I'm not saying I would change `toBeTruthy()` to `toBeInTheDocument()`, I'm just commenting because I found it interesting 😬

Reply:
I used `toBeTruthy` as it is "_native_" to the Jest's `expect`. However, `testing-library` does add `toBeInTheDocument` which makes it more clear, so I just changed it to use that. Thanks!